restpack_serializer.gemspec

```
@@ -18,8 +18,8 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency 'activesupport', ['>= 4.0.3', '< 6.0']
-  gem.add_dependency 'activerecord', ['>= 4.0.3', '< 6.0']
+  gem.add_dependency 'activesupport', ['>= 4.0.3', '< 7.1']
+  gem.add_dependency 'activerecord', ['>= 4.0.3', '< 7.1']
```

Our panoptes API is on rails 5.2 so we can bump past the 4.0 min
```suggestion
  gem.add_dependency 'activesupport', ['>= 5.2', '< 7.1']
  gem.add_dependency 'activerecord', ['>= 5.2', '< 7.1']
```